Yank/yank.py

```diff
@@ -88,6 +88,10 @@ def __init__(self, topology, ligand_atoms=None, solvent_atoms='auto'):
         self.solvent_atoms = solvent_atoms
         self.ligand_atoms = ligand_atoms
 
+        # Initialize regions
+        self._built_in_regions = ['ligand_atoms', 'receptor_atoms', 'solute_atoms', 'solvent_atoms', 'ion_atoms']
```

Comment:
It could make sense to define this as a private class-level constant since it's never changed.

Reply:
Is it not already a private class-level constant since its `self._name`? Or do you mean at `class name(); X=1, Y=2`? level

Reply:
I mean it should be a constant of the class, not of the object.